video_xblock/static/js/spec/studio-edit/transcript-manual-upload-spec.js

```diff
@@ -56,3 +56,68 @@ describe('Transcripts manual upload', function() {
         expect(validateTranscripts(e, $testTranscriptsBlock)).toBeTruthy();
     });
 });
+
+describe('Function "pushTranscript"', function() {
+    'use strict';
+    var newTranscriptAdded;
+    var transcriptsValue;
+    var testData = {
+        lang: 'en',
+        label: 'English',
+        url: 'testUrl',
+        source: 'manual',
+        oldLang: ''
+    };
+    var oldData = {
+        lang: 'en',
+        label: 'English',
+        url: 'otherTestUrl',
+        source: 'default'
+    };
+
+    afterEach(function() {
+        newTranscriptAdded = null;
+        transcriptsValue = null;
+    });
+
+
+    it('called with empty "transcriptsValue"', function() {
```

Comment:
First param of `it` function should actually describe what the test is checking for.
[Reference](https://jasmine.github.io/api/2.6/global.html#it)